src/idsk/components/header-web/_header-web.scss

```diff
@@ -0,0 +1,920 @@
+            @include govuk-focused-text;
+            box-shadow: none;
+          }
+
+          &:hover {
+            text-decoration: underline;
+          }
+        }
+
+        &--active {
+          background-color: $idsk-header-background-secondary;
+
+          .idsk-header-web__nav-list-item-link {
+
+            &:link,
+            &:hover,
+            &:visited {
+              color: $idsk-header-text-secondary;
+            }
+
+            &:focus,
+            &:active {
+              color: govuk-colour("black");
+
+              .idsk-header-web__link-arrow,
+              .idsk-header-web__link-arrow-mobile {
+                background-color: govuk-colour("black");
+              }
+            }
+
+            .idsk-header-web__link-arrow,
+            .idsk-header-web__link-arrow-mobile {
+              background-color: $idsk-header-text-secondary;
+            }
+          }
+          
+          .idsk-header-web__link-arrow,
+          .idsk-header-web__link-arrow-mobile {
+            transition: transform 0.2s;
+            transform: rotate(180deg);
+          }
+
+          .idsk-header-web__nav-submenu,
+          .idsk-header-web__nav-submenulite,
+          .idsk-header-web__nav-submenu-spacer{
+            display: block;
+          }
+
+          @include mq($until: desktop){
+            .idsk-header-web__nav-submenu-spacer{
+              display: none;
+            }
+
+            .idsk-header-web__nav-submenu {
+
+              .govuk-grid-row {
+                margin-left: -(govuk-spacing(6));
+                margin-right: -(govuk-spacing(6));
+              }
+
+              .govuk-grid-column-full {
+                margin-bottom: -1px;
+                padding: 0;
+              }
+              .govuk-grid-column-full:last-child {
+                margin-bottom: 0;
+              }
+            }
+          }
+        }
+      }
+    }
+
+    &--dark {
+      background-color: $idsk-header-background-secondary;
+
+      .idsk-header-web__link-arrow {
+        background-color: $idsk-header-text-secondary;
+      }
+
+      .idsk-header-web__nav-list-item-link {
+        color: $idsk-header-text-secondary;
+
+        &:focus,
+        &:active {
+          color: govuk-colour("black");
+
+          .idsk-header-web__link-arrow {
+            background-color: govuk-colour("black");
+          }
+        }
+      }
+    }
+
+    @include mq($until: desktop) {
+      border-top: none;
+
+      &-bar--buttons {
+        border-bottom: 2px solid govuk-colour("grey-3");
+      }
+
+      .idsk-header-web__main--buttons,
+      .idsk-header-web__main-action-search {
+        display: block;
+      }
+
+      .idsk-header-web__main-action-search {
+        margin: 0;
+
+        .govuk-input {
+          width: calc(100% - 44px);
+        }
+
+        .govuk-button {
+          float: right;
+        }
+      }
+      
+      &-list {
+        margin: govuk-spacing(3) -15px govuk-spacing(3) -15px;
```

Comment:
vieme na minusove marginy a paddingy pouzivat toto ? "-(govuk-spacing(6))"  lebo vidim ze to mas nizsie

Reply:
V pripade ze do marginu zapisujem viacero hodôt tak mi nezobere zaporny zapis iba tak ze tam napisem px. Ale aj tak som ho skratil na margin: govuk-spacing(3) -15px;